pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -538,7 +616,15 @@ private void handleSplitEvent(String serviceUnit, ServiceUnitStateData data) {
         }
     }
 
-    private void handleFreeEvent(String serviceUnit) {
+    private void handleDisableEvent(String serviceUnit, ServiceUnitStateData data) {
```

Comment:
This method was never used. Please remove it.

Reply:
Removed it.